buildSrc/src/main/groovy/airbyte-integration-test-java.gradle

```diff
@@ -44,8 +46,7 @@ class AirbyteIntegrationTestJavaPlugin implements Plugin<Project> {
             }
 
             maxHeapSize = '3g'
```

Comment:
Not sure what the effect of this would be, but maybe we can bump the maxHeapSize to 4g as well?

Reply:
I'm not sure it will actually make a difference (probably depends on your test suite but 3g is already a decent chonker of a heap), but it probably wouldn't actively hurt anything either

Reply:
Based in the test, we are using only 0'5g... so it's more than enough 🤣